src/main/java/com/google/javascript/clutz/Depgraph.java

```diff
@@ -135,6 +157,9 @@ private void collectFiles(boolean isRoots, List<List<?>> fileList) {
       } else {
         nonroots.add(fileName);
       }
+      if (!isGoogModule) {
```

Comment:
Name flag and the Set<String> consistently. Either both module or provide.

Reply:
Done.